controller/node_controller.go

```diff
@@ -281,8 +323,10 @@ func (nc *NodeController) syncNode(key string) (err error) {
 	if err != nil {
 		return err
 	}
+	removedNode := true
```

Comment:
It's better to call it `nodeManagerFound := false` since it's better to assume `true` for the most occasions rather than false.